### PR TITLE
Added french translations

### PR DESCRIPTION
**Pull request by Skeudwenn**

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -50,16 +50,16 @@ msgstr "Terminer le processus"
 #: src/Widgets/Headerbar.vala:26
 #, fuzzy
```
Comment (ryonakano) [inline po/fr.po:51]:
```suggestion
```

If you don't remove the `fuzzy` marks, your translation won't be applied to the UI; it means "this translation string is for the old source string and needs editing".

File: po/fr.po [hunk 2]
```diff
@@ -50,16 +50,16 @@ msgstr "Terminer le processus"
...
 msgid "End selected process"
-msgstr "Terminer le processus"
+msgstr "Terminer le processus sélectionné"
 
 #: src/Widgets/Headerbar.vala:30
 #, fuzzy
```
Comment (ryonakano) [inline po/fr.po]:
```suggestion
```

File: po/fr.po [hunk 3]
```diff
@@ -72,7 +72,7 @@ msgstr "Montrer icône du panneau"
 
 #: src/Widgets/Headerbar.vala:62
 msgid "Start in background:"
-msgstr ""
+msgstr "Démarrer en arrière-plan"
```
Comment (ryonakano) [inline po/fr.po]: I don't understand French, but don't you need the trailing colon?

**Comment by ryonakano** [comment]
Maybe some other French speakers like @NathanBnm can take a look at this regarding to the grammar or wording of these translations?

File: po/fr.po (hunk 1 above)
Comment (Skeudwenn) [reply po/fr.po:51]: Thanks, didn't know that !

File: po/fr.po (hunk 3 above)
Comment (Skeudwenn) [reply po/fr.po]: Oups, my bad !

File: po/fr.po (hunk 1 above)
Comment (ryonakano) [reply po/fr.po:51]: You don't seem to remove this line.

Comment (Skeudwenn) [reply po/fr.po:51]: Well, I'm not truly satisfied by the french translation. This is why I didn't remove the fuzzy comment.

**Comment by NathanBnm** [comment]
Thanks @ryonakano I'm going to check

File: po/fr.po [hunk 4]
```diff
@@ -50,16 +50,15 @@ msgstr "Terminer le processus"
 #: src/Widgets/Headerbar.vala:26
 #, fuzzy
 msgid "End selected process"
-msgstr "Terminer le processus"
+msgstr "Terminer le processus sélectionné"
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
msgstr "Mettre fin au processus sélectionné"
```

File: po/fr.po (hunk 1 above)
Comment (NathanBnm) [reply po/fr.po:51]: @Skeudwenn I just left a new comment about this.

File: po/fr.po (hunk 4 above)
Comment (NathanBnm) [inline po/fr.po:53]:
```suggestion
msgstr "Mettre fin au processus sélectionné"
```

**Comment by NathanBnm** [comment]
I have no access to comment unchanged strings in this PR. But these are mistranslated. Should I open a new PR and include these changes?

**Comment by NathanBnm** [comment]
@Skeudwenn are you speaking French natively?

**Comment by Skeudwenn** [comment]
@NathanBnm Yes !

**Comment by ryonakano** [comment]
@NathanBnm Thank you for your review!

GitHub does not seem to support commenting on lines that have no changes in a PR, so I guess you need to copy and paste the current translation sentences and your suggestions here or make a new PR against https://github.com/Skeudwenn/monitor/tree/french-translations.

**Comment by NathanBnm** [comment]
@Skeudwenn I think you can close this PR now

**Comment by ryonakano** [comment]
Hmm, I think we can keep this PR alive. Although #130 might override this PR, I think we can merge both of Nathan's and Skeudwenn's contributions.

**Comment by stsdc** [comment]
Thanks @Skeudwenn!
Thanks @ryonakano  for reviews!

**Pull request merged**